### PR TITLE
.NET 5 and .NET Core naming doc

### PR DESCRIPTION
**Pull request by tdykstra**
@BillWagner @IEvangelist Please review

### CODE REVIEW

File: Contribute/TOC.yml [hunk 1]
```diff
@@ -66,6 +66,8 @@
           href: dotnet/dotnet-voice-tone.md
         - name: Pull request review process
           href: dotnet/dotnet-pr-review.md
+        - name: .NET 5 and .NET Core naming
```
Comment (IEvangelist) [inline Contribute/TOC.yml:69]:
```suggestion
        - name: .NET naming in docs
```

File: Contribute/dotnet/net-core-5-naming.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+title: "How to refer to .NET Core / 5+ in docs"
```
Comment (IEvangelist) [inline Contribute/dotnet/net-core-5-naming.md:2]:
```suggestion
title: "How to refer to .NET in docs"
```

File: Contribute/dotnet/net-core-5-naming.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+description: "Learn how to refer to .NET Core, .NET 5, and later versions in docs."
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+ms.date: 01/28/2021
+---
+# How to refer to .NET Core / 5+ in docs
```
Comment (IEvangelist) [inline Contribute/dotnet/net-core-5-naming.md:9]:
```suggestion
# How to refer to .NET in docs
```

File: Contribute/dotnet/net-core-5-naming.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+## Recommended terminology
+
+**In Doc Title/H1:**
```
Comment (IEvangelist) [inline Contribute/dotnet/net-core-5-naming.md:13]:
```suggestion
**In doc title and first heading (h1):**
```

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Comment by tdykstra** [comment]
Thanks for the review, @IEvangelist 

#sign-off

**Pull request merged**